docs/cvl/expr.md

```diff
@@ -365,6 +365,15 @@ while verifying the rule, and will provide a separate verification report for
 each checked method.  Rules that use this feature are referred to as
 {term}`parametric rule`s.
 
+Another possible way to have the Prover consider options for a function is by
+prefixing the function name with an `address` typed variable. In this case the
+Prover will consider every possible contract in the scene that implements a
+function that matches the signature provided by the call (if no such function
+exists in the scene the prover will fail with an error). If there are no
+relevant constraints on the `address` variable the prover may also consider the
```

Comment:
_"the prover may also consider"_? Either it considers it or not, do not use "may"